vertical-pod-autoscaler/e2e/v1/full_vpa.go

```diff
@@ -101,7 +101,7 @@ var _ = FullVpaE2eDescribe("Pods under VPA", func() {
 		rc.ConsumeCPU(600 * replicas)
 		err = waitForResourceRequestInRangeInPods(
 			f, pollTimeout, metav1.ListOptions{LabelSelector: "name=hamster"}, apiv1.ResourceCPU,
-			ParseQuantityOrDie("500m"), ParseQuantityOrDie("900m"))
+			ParseQuantityOrDie("500m"), ParseQuantityOrDie("1000m"))
```

Comment:
That is 1, right?

Reply:
Yes.
Is this a suggestion to change it?

Reply:
IMHO is easier to read if we have the same units on each side of the range.

Reply:
IMHO 1000m looks weird, but I won't block.